test/bin/scenario.sh

```diff
@@ -170,11 +171,17 @@ prepare_kickstart() {
     local output_file
     local vm_hostname 
     local fips_command=""
+    local -r hostname=$(hostname)
+    local mirror_cert=""
 
     full_vmname="$(full_vm_name "${vmname}")"
     output_file="${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${vmname}/kickstart.ks"
     vm_hostname="${full_vmname/./-}"
 
+    if [ "${ENABLE_MIRROR}" ]; then
+        mirror_cert="$(awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' /etc/pki/ca-trust/source/anchors/mirror-registry.pem)"
```

Comment:
Why do we need this? 
Can you add a comment in any case if we leave this code?

Reply:
explained above, added a comment.